<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="resetButton">
        <property name="text">
         <string>Reset Simulation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="frictionButton">
        <property name="text">
         <string>Friction ON/OFF</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="gravityButton">
        <property name="text">
         <string>Alter Gravity</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actiontaper_ici">
   <property name="text">
    <string>taper ici</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
